src/cosmic_ray/operators/binary_operator_replacement.py

```diff
@@ -54,12 +54,12 @@ def examples(cls):
 
 
 # Parent types of operators which indicate that the operator isn't binary.
-_NON_BINARY_PARENTS = set((
```

Comment:
Please please please stop putting spurious, unrelated changes into your PRs. My time for processing these is very limited, and whenever you do this I have to spend some time figuring out *why* you've done it. When the answer is "for no reason related to the actual change", I get frustrated.